src/front/zsharp/TODO

```diff
@@ -0,0 +1,70 @@
+- casts
```

Comment:
Do you want to keep this around? Just checking to make sure it is intentional.

Reply:
yeah, I still need to add some widening casts to the stdlib, so I'll leave this one.